src/process_data.c

```diff
@@ -101,13 +102,11 @@ ssize_t process_request_set( const char *filename, wrp_msg_t *req, const char *m
             fclose(file_handle);
         }
         free(md5_string);
+        process_schedule_data(req->u.req.payload_size, req->u.req.payload);
```

Comment:
This should happen & return a success before we write the output to the file.  If `process_schedule_data()` doesn't return success/failure due to processing the file, that should change.

Reply:
Got it. Done.